### PR TITLE
Add: Sorting feature for jokes

### LINKED ISSUE #91: feature: Get jokes by Filter
add code for Get jokes by Filter

- [ ] http get (filter model?)

### PR DESCRIPTION
**Pull request by HellHunterMax**
Ok much cleaner imo. thanks for the tips!

Closes #91

### CODE REVIEW

File: Server/JokesOnYou.Web.Api/Controllers/JokesController.cs [hunk 1]
```diff
@@ -39,6 +40,14 @@ public async Task<ActionResult<IEnumerable<JokeReplyDto>>> GetAllJokesAsync()
             return Ok(jokeDtos);
         }
 
+        [AllowAnonymous]
+        [HttpGet("Results")]
+        public async Task<ActionResult<IEnumerable<JokeReplyDto>>> GetFilteredJokesAsync([FromQuery]JokesFilterDto jokesFilter)
```
Comment (chrisK00) [inline Server/JokesOnYou.Web.Api/Controllers/JokesController.cs]: We dont need a separate endpoint for filtering, filters in a query string are optional by default.

Comment (HellHunterMax) [reply Server/JokesOnYou.Web.Api/Controllers/JokesController.cs]: cool I'll replace the GetAllJokes.

**Review by chrisK00 — APPROVED** [review]
Well done! Time to search for Nico's jokes

**Comment by Niisc** [comment]
> 
> 
> Well done! Time to search for Nico's jokes

Oh no

**Pull request merged**